suites/create.js

```diff
@@ -308,6 +308,101 @@ export function runDataIntegrityProofFormatTests({
           }
         }
       });
+      describe('Algorithms', function() {
+        describe('Add Proof', function() {
+          it('Whenever this algorithm encodes strings, ' +
+            'it MUST use UTF-8 encoding.',
+          function() {
+            for(const proof of proofs) {
+              chai.expect(proof.proofValue.isWellFormed()).to.be.true;
+            }
+          });
+          it.skip('If the algorithm produces an error, the error MUST be ' +
+          'propagated and SHOULD convey the error type.',
+          function() {});
```

Comment:
Can you leave a comment about why this is being skipped?

We also now support `this.test.cell.skipMessage` being set so pending test icons get a mouse over.

Reply:
@PatStLouis @BigBlueHat is correct here, we now have a `skipMessage` you can to describe the reason the test is being skipped.

Reply:
I've implemented skipMessage